src/py-opentimelineio/opentimelineio/console/otiotool.py

```diff
@@ -380,6 +413,10 @@ def parse_arguments():
 
     args = parser.parse_args()
 
+    # At least one of these must be specified
+    if not any([args.input, args.list_versions]):
```

Comment:
Relating to my previous comment, should they be mutually exclusive?

Reply:
Fixed.